src/index.html

```diff
@@ -8,6 +8,107 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+    </div>
+    <div class="stars stars--2">
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+    </div>
+    <div class="stars stars--3">
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+    </div>
+    <div class="stars stars--4">
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star">
+
+      </div>
+    </div>
+    <div class="stars stars--5">
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
+
+      </div>
+      <div class="stars__star is__active">
```

Comment:
```suggestion
      <div class="stars__star">
```
in all places this class redundant
about amount active stars should be responsible parent modifier (stars--0, stars--1 ...)